rebuild/src/addon.h

```diff
@@ -0,0 +1,177 @@
+#ifndef BLST_TS_ADDON_H__
```

Comment:
`ifndef` directives are a bit error prone, especially when some renaming happened. Modern compilers support `#pragma once` that should be preferred.